test/common/http/common.h

```diff
@@ -39,8 +39,8 @@ class CodecClientForTest : public Http::CodecClient {
  * Mock callbacks used for conn pool testing.
  */
 struct ConnPoolCallbacks : public Http::ConnectionPool::Callbacks {
-  void onPoolReady(Http::StreamEncoder& encoder,
-                   Upstream::HostDescriptionConstSharedPtr host) override {
+  void onPoolReady(Http::StreamEncoder& encoder, Upstream::HostDescriptionConstSharedPtr host,
+                   ABSL_ATTRIBUTE_UNUSED const StreamInfo::StreamInfo& info) override {
```

Comment:
nit: just make the variable unnamed instead of `ABSL_ATTRIBUTE_UNUSED`

Reply:
done.